pint_server/__init__.py

```diff
@@ -16,7 +16,7 @@
 # you may find current contact information at www.suse.com
 
 # NOTE(gyee): must update the version here on a new release
-__VERSION__ = '2.0.6'
+__VERSION__ = '2.0.7'
```

Comment:
Agreed but #105 is incomplete right now as still need to implement the validation that all regions either have the changeinfo or no regions have it part, so it will probably get tweaked/fixed to bump the version again once I get that work done.

Reply:
Hopefully will get time soon to sort out some sort of automatic version bump management, e.g. as part of a post merge handler, if we detect that the pint_server.py implementation has changed, automatically bump the version, merge the bump and create the new version tag.